docs/version-specific/easyblocks.rst

```diff
@@ -223,6 +227,7 @@
   * EB_Gctf
   * EB_libsmm
   * EB_Modeller
+  * EB_MOOSE
```

Comment:
?

Reply:
Oh, that's a local one, totally my bad... Nice catch! Fixing it now...